src/components/Modal/Modal.js

```diff
@@ -50,7 +50,8 @@ class Modal extends Component {
       className,
```

Comment:
This should be removed because `className` is added on line 54.

```suggestion
```

Reply:
good call